WorkloadManagementSystem/Service/PilotsLoggingHandler.py

```diff
@@ -0,0 +1,85 @@
+""" PilotsLoggingHandler is the implementation of the PilotsLogging service
+
+    The following methods are available in the Service interface
+
+    addPilotsLogging()
+    getPilotsLogging
+    setPilotsUUIDtoIDMapping()
+    addPilotsUUID()
+    deletePilotsLogging()
+
+"""
+
+__RCSID__ = "$Id: $"
+
+from DIRAC.WorkloadManagementSystem.DB.PilotsLoggingDB import PilotsLoggingDB
+from DIRAC.Resources.MessageQueue.MQCommunication import createConsumer
+
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC import S_OK
+
+
+class PilotsLoggingHandler( RequestHandler ):
+  """Server side functions for Pilots Logging service"""
+
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    """Initialization of Pilots Logging service
+    """
+    cls.consumersSet = set()
+    cls.pilotsLoggingDB = PilotsLoggingDB()
+
+    result = createConsumer( "lbvobox50.cern.ch::Queue::test", callback = cls.consumingCallback ) # XXX hardcoded URI
```

Comment:
The definition of the MQ to use should be a parameter. As a start I suggest to take it from the service CS section via
```
self.srv_getCSOption("PilotsLoggingQueue")
```